File: testprojects/csharp/benchmark/domain/shortcharacter/twitter/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.socialmedia;

namespace CSharpBenchmark.domain.shortcharacter.twitter
{
    //Functional requirement 3.1.1
    //Test case 36: Class domain.shortcharacter.twitter.Account may only have a dependency with class infrastructure.socialmedia.SocialNetwork
    //Result: TRUE
    public class Account
    {
        public Account()
        {
            //FR5.1
            new SocialNetwork();
        }
    }
}

File: testresources/java/benchmark_2012/domain/shortcharacter/identica/Account.java
package domain.shortcharacter.identica;

import infrastructure.asocialmedia.ASocialNetwork;
//Functional requirement 3.1.1
//Test case 37: Class domain.shortcharacter.identica.Account may only have a dependency with class infrastructure.asocialmedia.ASocialNetwork
//Result: FALSE
public class Account{
	public Account(){
		//FR5.1
		new ASocialNetwork();
	}
}